test/server.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+const gateway = require('express-gateway');
+
+gateway()
+  .load(path.join(__dirname, 'config'))
+  .run();
```

Comment:
I don't see any test here except it runs. 
Minimal variant is to run real server + fake backend server and ensure that fake backend is recieving correct url
same for redirect

Reply:
Definitely - as I wrote in the pull request text, I'm figuring out how to test but more importantly - I'm missing a way to shut down the gateway programmatically. I can see from the gateway repo you guys have an utility to fork and kill the process and I'm not entirely sure whether I should just copy paste that code here or eventually adding a shutdown function to the gateway. What's your opinion?

Reply:
Addressed!